scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/MerlinService.java

```diff
@@ -84,12 +84,13 @@ void ensurePlanExists(final PlanId planId)
     throws IOException, NoSuchPlanException, MerlinServiceException;
 
     /**
-     * Gets existing simulation results for current plan if they exist and are suitable for scheduling purposes (current revision, covers the entire planning horizon)
+     * Gets existing simulation results for current plan if they exist and are suitable for scheduling purposes (current
+     * revision, covers the entire planning horizon)
      * These simulation results do not include events and topics.
      * @param planMetadata the plan metadata
-     * @return simulation results, optionally
+     * @return simulation results and its dataset ids, optionally
```

Comment:
Minor, nonblocking, clarity: There's been confusion about `dataset id` vs `sim dataset id` in the past, so I'd prefer this comment not to group them as "dataset ids". 
Nonblocking as technically the type is called `DatasetIds`.
```suggestion
     * @return optionally: simulation results, its dataset id, and its simulation dataset id
```

Reply:
To clarify, did you intend to update the comment? GH shows the original comment, but you put a :+1:.

Reply:
changed the comment